plugins/events-backend-module-sqs/config.d.ts

```diff
@@ -0,0 +1,60 @@
+/*
+ * Copyright 2022 The Backstage Authors
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+export interface Config {
+  events?: {
+    /**
+     * List of publisher-specific options and attributes.
+     *
+     * @visibility backend
+     */
+    providers?: {
+      /**
+       * events-backend-module-sqs plugin configuration.
+       *
+       * Contains a record per topic for which AWS SQS
+       * should be used as source of events.
+       *
+       * @visibility backend
+       */
+      awsSqs?: Record<
```

Comment:
oh and should these have a mapping from external to internal topic names?

Reply:
makes sense to add `topics` as key in-between.

So far I still lean towards using topic names as keys instead of as property inside of an array. Arrays are harder to override using env vars, etc.

> oh and should these have a mapping from external to internal topic names?

Which external mapping are you referring to? At the SQS queue, you could consider the queue name sort of as a topic. However, we don't really care what is used externally. We just pull messages from the queue and pass them on to the assigned topic.

Reply:
I've added topics as sub-key for better extendability as suggested. I sticked to Record instead of Array still though. Let me know your thoughts, of course.